hw_10.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+// Task 1
+$x = 'Hello World';
+
+// Task 2
+
+$x = 5;
+$y = 20;
+echo $x + $y;
+
+// Task 3
+
+$str = 'Привет мир!';
+echo mb_strlen($str);
+
+// Task 4
+
+$oldText = "Learn HTML!";
+echo str_replace("HTML","PHP",$oldText);
```

Comment:
не забываем про пробелы после запятой
```suggestion
echo str_replace("HTML", "PHP", $oldText);
```